btr-web/btr-main-app/components/individual-person/summary-table/CessationDateEntry.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+import { SiSchemaType } from '~/utils/si-schema/definitions'
+
+const props = defineProps<{ individual: SiSchemaType, index: number }>()
+
+const { allEditableSIs }: {
+  allEditableSIs: Ref<SiSchemaType>
+} = storeToRefs(useSignificantIndividuals())
+
+const cessationDate: Ref<string | undefined> = ref(undefined)
+const submitDate = () => {
+  if (!cessationDate.value) {
+    return
+  }
+  // update individual
+  allEditableSIs.value[props.index].ui.showCeaseDateInput = false
+  useSignificantIndividuals().filingCeaseSI(props.index, cessationDate.value)
+}
+const cancel = () => {
+  allEditableSIs.value[props.index].ui.showCeaseDateInput = false
+}
+</script>
+
+<template>
+  <BcrosSection
+    :border="true"
+    rounded-top
+    rounded-bottom
+    :section-title="$t('labels.cessationDate')"
+  >
+    <div class="flex-col w-full">
+      <BcrosInputsDateSelect
+        name="cessationDate"
+        :max-date="new Date()"
```

Comment:
I think we need to set the `:min-date` here so the cessation date cannot be earlier than the start date.

Reply:
I think we have to let them put in anything even if its wrong. We went through this previously with the dates in the form too

Reply:
Oh actually nvm I am wrong -- will add this

Reply:
okay added that in